independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/handlers/ClientSendRequestHandler.java

```diff
@@ -384,10 +383,10 @@ public void handle(AsyncResult<Buffer> ar) {
                 .onFailure(new Handler<>() {
                     @Override
                     public void handle(Throwable failure) {
-                        if (failure instanceof HttpClosedException) {
+                        if (failure instanceof RuntimeException) {
```

Comment:
Are you sure we should catch all `RuntimeException` here? The comment makes is seem like it should only be the two exceptions mentioned?

But maybe I'm being too cautious, I will defer to @geoand on that.

Reply:
Yes, I was thinking about that, but I also didn't want to end up with a huge list of exceptions there.